### DISCUSSION TITLE
Sponsorhip?

### DISCUSSION THREAD

**Discussion by Fleshgrinder** [actions/actions-runner-controller#1242] [General]
@mumoshu @summerwind @toast-gear and who else is representing this project. Thanks for your collaboration in https://github.com/actions-runner-controller/actions-runner-controller/issues/1135 and I'm looking forward to work on other things together with you (e.g. https://github.com/actions-runner-controller/actions-runner-controller/issues/1137).

Given that we are relying on this project for now I wanted to ask you if it is possible to sponsor your work on this project somehow. I do not know how to reach out to you, hence the discussion. However, you can contact me privately via email at fleshgrinder@users.noreply.github.com so we can discuss possible sponsorship to ensure that ARC is a healthy and sustainable product that enterprise users like myself can build on top of. 😊💵

**Comment by mumoshu** [comment]
@Fleshgrinder Hey! Thank you for your efforts and support 🙏

I'd definitely appreciate any sponsorships. I have a GitHub Sponsors page so please go ahead and choose any tier that makes sense to you.
I'm available via the email address I use for git commits, and also @mumoshu at https://slack.sweetops.com/. Contact me with either channel!

// I tried your dummy github email but it didn't work as it's a dummy :smile:

**Comment by Fleshgrinder** [comment]
Apparently GitHub changes these email addresses, I didn't even notice. 🤣 Now it’s 1059453+Fleshgrinder@users.noreply.github.com with an ID in front.[^1] I am actually registered since longer than 2017, but I for sure updated my email settings at some time and thus got a new one.

Anyway, I will get in touch. 😉

[^1]: https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-user-account/managing-email-preferences/setting-your-commit-email-address#about-commit-email-addresses

**Comment by mumoshu** [comment]
Hey! I just confirmed the noreply address doesn't work regardless of the user id prefix. If I don't miss anything, it's intended to be used internally between you and GitHub and not a real email address.
Anyway, I hope you find your way to my email or me on the sweetops slack! Thanks.
